README.md

```diff
@@ -11,47 +11,41 @@
 
 This documentation describes the process to apply and validate an IoT Device as FIWARE Ready.
 
-To join FIWARE-ready IoT Devices Program, you need to follow a series of steps, including an simple validation of your
-product (implementation), where you will connect it to a FIWARE public instance and you will perform a series of tests:
+To be featured as a FIWARE-ready IoT Device, you need to follow some steps, including a simple validation of your
```

Comment:
Not sure if device has to be in uppercase...

Reply:
fixed  2e0c06a